build.gradle

```diff
@@ -44,16 +42,12 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-json'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
+	implementation 'org.springframework.cloud:spring-cloud-aws-messaging:2.2.6.RELEASE'
```

Comment:
Doesn't require this as doesn't listen to queues.

We only actually need https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-sns for this...

Reply:
I'll raise this against Casework too then which is what I cribbed from...